### DISCUSSION TITLE
I'm unable to use the software: in my command line "qpdf"  is not recognised

### DISCUSSION THREAD

**Discussion by MilosBranko** [qpdf/qpdf#951] [General]
I downloaded qpdf and installed it. Then I started power shell and I tried to use the program with a command with following syntax:
PS C:\Program Files\qpdf 11.3.0\bin> qpdf --decrypt input.pdf output.pdf
but it doesn't work, I get only 
>>
Please, let me know what I mistake and possibly what a path and/or command should I use instead. 
Thanks

**Comment by jberkenbilt** [comment]
Your post doesn't include what you got, at least not that I can see. The command is fine. If the file has a password, you'll need to specify it with `--password=whatever`.

**Comment by MilosBranko** [comment]
Hi there,

thank you very much for your kind and prompt reply.
What you couldn’t see in my previous message was just this:
    >> 
Anyway, I tried again but without success, I think I misunderstood or mistook something, here is what I got this time:

C:\Program Files>qpdf  C:\Users\Stefano\Documents\_DOC_SCARICATI\Assicurazioni\Allianz\2019-01-14 Prev polizza Casa rivista €252 6m.pdf  C:\Users\Stefano\Documents\_DOC_SCARICATI\Assicurazioni\Allianz\2019-01-14 Prev polizza Casa rivista.pdf

"qpdf" non è riconosciuto come comando interno o esterno, un programma eseguibile o un file batch.
(Qpdf is not recognized as an internal or external command, an executable program or a file batch)

 Please, let me know what you think, thank you.

 

Da: Jay Berkenbilt ***@***.***> 
Inviato: lunedì 17 aprile 2023 19:54
A: qpdf/qpdf ***@***.***>
Cc: MilosBranko ***@***.***>; Author ***@***.***>
Oggetto: Re: [qpdf/qpdf] I'm unable to use the software: in my command line "qpdf" is not recognised (Discussion #951)

 

Your post doesn't include what you got, at least not that I can see. The command is fine. If the file has a password, you'll need to specify it with --password=whatever.
